app/types/prison_type.rb

```diff
@@ -0,0 +1,5 @@
+class PrisonType < ActiveModel::Type::Value
+  def cast(value)
```

Comment:
If input == output is casting necessary in this instance?

Reply:
Yeah I'm affraid that's how active model attributes work.

💡  **However** nothing prevents us from writing attr_accessor method and get rid of this nonsense!

Reply:
So I'll amend that